Common/itkComputeDisplacementDistribution.hxx

```diff
@@ -519,7 +519,6 @@ ComputeDisplacementDistribution<TFixedImage, TTransform>::ComputeUsingSearchDire
   Jgg.Fill(0.0);
   std::vector<double> JGG_k;
   double              globalDeformation = 0.0;
-  JacobianType        jacjjacj(outdim, outdim);
```

Comment:
@mstaring Do you agree that this unused parameter (`jacjjacj`) may simply be removed? It allocates dynamic memory, so removing the variable _might_ improve the performance  😺 More importantly though, was it _meant_ to be used at all?

Reply:
For the record, Marius just double-checked, and thought this commit should be OK 👍